src/include/processor/operator/persistent/copy_rel.h

```diff
@@ -48,14 +48,14 @@ class CopyRelSharedState {
     }
     inline void updateRelsStatistics() { relsStatistics->setNumTuplesForTable(tableID, numRows); }
 
-    inline std::shared_ptr<FactorizedTable> getFTable() { return fTable; }
```

Comment:
Why move the get function and expose this as public here?

Reply:
We will try to solve this with comment above together.